pynetinstall/flash.py

```diff
@@ -219,7 +219,10 @@ def run(self, info: InterfaceInfo) -> None:
         self.logger.debug("Sending the offer to flash")
         try:
             self.state = [0, 0]
-            self.do(b"OFFR\n\n", b"YACK\n")
+            if info.magic:
+                self.do(b"OFFR\n"+info.magic+bytes([0x0A, 0x0A, 0x0A, 0x00]), b"YACK\n")
```

Comment:
any reason why we don't always send that? also, 0x0a == '\n', so we can just do

```suggestion
                self.do(b"OFFR\n%b\n\n\n\0" % info.magic, b"YACK\n")
```

Reply:
Now that you mention it, I would say that other models also expect the license key, so probably safe always sending it in the OFFR